ui_framework/components/color_picker/color_picker.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import { ChromePicker } from 'react-color';
+
+export class KuiColorPicker extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showColorSelector: false,
+    };
+  }
+
+  closeColorSelector = () => {
+    this.setState({ showColorSelector: false });
+  };
+
+  toggleColorSelector = () => {
+    this.setState({ showColorSelector: !this.state.showColorSelector });
+  };
+
+  handleColorSelection = (color) => {
+    this.props.onChange(color.hex);
+  };
+
+  render() {
+    const color = this.props.color || '#ffffff';
```

Comment:
That makes sense. Though now that you mention it, I think it makes sense for the owner to pick a default. Since this component has no knowledge of its context, it doesn't really know whether a sensible default would be white, pink, or fuchsia (which, now that I google it, is the same color as pink... go figure).